strutil/strutil.go

```diff
@@ -22,13 +22,35 @@ package strutil
 import (
 	"fmt"
 	"io"
+	"math/rand"
 	"sort"
 	"strconv"
 	"strings"
+	"time"
 	"unicode"
 	"unicode/utf8"
 )
 
+func init() {
+	// golang does not init Seed() itself
```

Comment:
I would personally not see this as a surprise. (as in this is normal for me to have to do this in order to be in control of how random the pseudo generator appear.

Reply:
(a bit of a pedantic/nitpicking comment, sorry!) What I like about `randutil` (see my other comment) is that it's less surprising to see something messing with the RNG seed on import when it's called `randutil` than when it's called `strutil`.

Reply:
I don't think we should ever mess with something like that just from importing. This should be an app-level concern. Just importing something like this shouldn't have a non-obvious side effect.